styles.scss

```diff
@@ -0,0 +1,251 @@
+$backgroundColor: #f5f5f5;
+$darkBackgroundColor: #949494;
+$buttonColor: #5E7FC1;
+$marginAuto: 0 auto;
+$mainPadding: 30px;
+$basicFont: 1rem;
```

Comment:
Chválím použití relativních jednotek pro fonty, určitě by se ale líbily i ostatním místům, kde zatím sedí pixely. :thinking: Pěkný článek (nevím, jestli jsem ho už nesdílela?) je o tom [tady](https://www.vzhurudolu.cz/prirucka/jednotky). Hodně ti pomůžou s responsivním designem.